modules/GameModule.mjs

```diff
@@ -0,0 +1,175 @@
+import Counter from '../modules/Counter.mjs'
+import EnemyFactory from '../modules/EnemyFactory.mjs'
+
+const enemyFactory = EnemyFactory()
+const counter = Counter()
+
+const GameModule = () => {
+
+   const _gameState = {
+    initialSeconds:0,
+    secondsLeft: 0,
+    currentLevel: 1,
+    enemies:[],
+    enemiesLeft: 0,
+    score: 0,
+    gameOn: false,
+    gameOver: false,
+    gameStart: false,
```

Comment:
as you also said, if you don't use all the variables you can delete the ones that aren't in use - and also note that gameOn and gameOver are the same but just opposite